inc/cliopts.php

```diff
@@ -447,7 +447,7 @@ class Doku_Cli_Opts_Error {
     var $code;
     var $msg;
 
-    function Doku_Cli_Opts_Error($code, $msg) {
+    function __construct($code, $msg) {
```

Comment:
the change is fine, but the class itself is deprecated anyway, so maybe we can remove it (have to check the date)